### DISCUSSION TITLE
GetExecutablePath with Version 11

### DISCUSSION THREAD

**Discussion by fletcher-arnold** [hardkoded/puppeteer-sharp#2293] [Q&A] [answered]
Using the following code:
BrowserFetcher browserFetcher = new(new BrowserFetcherOptions {Path = @"C:\Puppeteer"});
await browserFetcher.DownloadAsync(Chrome.DefaultBuildId);

This results in a path of "C:\Puppeteer\Chrome\Win64-115.0.5790.102\chrome-win64\chrome.exe"

But how can I get this path from code?
Previously I used:
path = browserFetcher.GetExecutablePath(BrowserFetcher.DefaultChromiumRevision);
but this no longer builds under v11

Thanks for any help...

**Comment by kblok** [comment, marked as answer]
Hey @fletcher-arnold. Let me know if this helps:

```
browserFetcher.GetInstalledBrowsers().First(browser => browser.BuildId == Chrome.DefaultBuildId).GetExecutablePath();
```

**Reply by fletcher-arnold** [reply]
Hi Darío
That's exactly what I was looking for. Thank you so much for your help :-)
Fletcher

**Reply by anaya-upadhyay** [reply]
This helped. Thanks @kblok
